package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -584,10 +584,9 @@ def _parse_box(self, header):
 
 
 class LammpsDumpParser(TopologyReaderBase):
-    """Parses Lammps ascii dump files in 'atom' format
-
-    Only reads atom ids.  Sets all masses to 1.0.
-
+    """Parses Lammps ascii dump files in 'atom' format.
+    Sets all masses to 1.0.
+    .. versionchanged:: 2.0.0
```

Comment:
Does this format correctly? Normally we have two blank lines above `.. versionadded`